lib/tether/toolbox.go

```diff
@@ -219,25 +219,30 @@ func (t *Toolbox) halt() error {
 
 // externalIP attempts to find an external IP to be reported as the guest IP
 func (t *Toolbox) externalIP() string {
-	netif, err := net.InterfaceByName("client")
+	l, err := netlink.LinkByName("client")
```

Comment:
We should probably consider moving "client" to a constant, as its now referenced 6 times within this routine and likely many places elsewhere in vic.